### ISSUE TITLE
Tag 1.1: Unidentified segfault after 8+ hours

### ISSUE THREAD

**Issue by gammy** [gammy/nes_gamepad#2]
After some 8+ hours of continuous use, the program, running in daemon mode, dumped core:

```
Nov 12 22:22:54 hypatia kernel: uinput-nes[7871]: segfault at 4e3904fa4 ip 0000000000401c8f sp 00007fffe3904f40 error 4 in uinput-nes[400000+4000]
Nov 12 22:22:54 hypatia systemd-coredump[8410]: Process 7871 (uinput-nes) dumped core.
```

I don't know if the length of the session was to blame, or if something else happened.
I'm still trying to reproduce it.

**Comment by gammy** [comment]
Three corruption-prone bugs have been found and corrected since 1.1 (Commit 4dde73b39a7a157c0400f0eac00f6895c1ede7e2, c55a031e4317554d3cdd59e148efdf9567787321); these are both likely candidates for this segmentation fault, however this issue will remain open.

**Comment by gammy** [comment]
No it won't. :p
